tests/end2end/test_public_logging_in.py

```diff
@@ -171,6 +173,8 @@ def test_sees_error_message_if_username_doesnt_exist(user, testapp):
     res = form.submit()
     # Sees error.
     assert _('Unknown username/email') in res
+    # Also traceable in Nginx logs:
+    assert res.headers['X-Username'] == 'unknown@nowhere.com'
```

Comment:
Maybe we should use `@example.com` instead? Seems like `nowhere.com` is a real domain?